daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -122,6 +122,22 @@ commandParser = subparser $ fold
         <*> (ScriptOptions <$> many (strOption (long "script-option" <> metavar "SCRIPT_OPTION" <> help "Pass option to DAML script interpreter")))
         <*> stdinCloseOpt
 
+    navigatorFlag =
+        -- We do not use flagYesNoAuto here since that doesn’t allow us to differentiate
+        -- if the flag was passed explicitly or not.
```

Comment:
Should we change `flagYesNoAuto` to have the behaviour we need here in a followup PR?

Reply:
There are two things that are different here:

1. We need to be able to check if the flag was passed or not.
2. We want `true` and `false` to work as well. 

There are quite a few other uses of `flagYesNoAuto` that don’t need this so for now I would leave it as is. I somewhat wonder if the `flagYesNoAuto` thing is a good idea anyway and we should just switch to `true` `false` and no `auto` everywhere but I’m not touching that for now.